src/App.tsx

```diff
@@ -1,24 +1,46 @@
 import React from 'react';
-import logo from './logo.svg';
+import Header from './Header'
+import Params from './Params'
+import Calendar from './Calendar'
 import './App.css';
+import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+
+const drawerWidth = 240;
+const useStyles = makeStyles((theme: Theme) => 
```

Comment:
styles object could be exported to separate file as it's not directly part of this component

Reply:
done